tfx_bsl/beam/run_inference.py

```diff
@@ -413,13 +415,31 @@ def _prepare_instances(
         if attr_name is None:
           continue
         attr = getattr(feature, attr_name)
-        values = cls._parse_feature_content(attr.value, attr_name,
-                                            cls._sending_as_binary(input_name))
+        values = self._parse_feature_content(attr.value, attr_name,
+                                             self._sending_as_binary(input_name))
         # Flatten a sequence if its length is 1
         values = (values[0] if len(values) == 1 else values)
         instance[input_name] = values
       yield instance
 
+  def _prepare_instances_serialized(
+      self, elements: List[tf.train.Example]
+  ) -> Generator[Mapping[Text, Any], None, None]:
+    """Prepare instances by base64 encoding serialized examples"""
+    for example in elements:
+      instance = {
```

Comment:
Directly yield {'b64': base64.b64encode(example.SerializeToString()).decode()} ?

Reply:
@rose-rong-liu Do you suggest not to have input_name? I just want to double check as it was in the design doc.

Reply:
Yes, since input name is not necessary on cloud API, and it would require users to give us the input name.